jaas/src/main/java/com/griddynamics/jagger/jaas/service/impl/TestEnvironmentServiceImpl.java

```diff
@@ -100,7 +94,7 @@ else if (!isEqualCollection(testEnvToUpdate.getLoadScenarios(), newTestEnv.getLo
         }
 
         if (testEnvToUpdate.getRunningLoadScenario() != newTestEnv.getRunningLoadScenario()) {
-            testEnvToUpdate.setRunningLoadScenario(getNewRunningLoadScenario(newTestEnv, testEnvToUpdate));
+            testEnvToUpdate.setRunningLoadScenario(newTestEnv.getRunningLoadScenario());
```

Comment:
do you mean I can't update more then 1 field in a single transaction? or what is it related with?